function-aws-custom-runtime/src/main/resources/META-INF/native-image/io/micronaut/function/aws/runtime/native-image.properties

```diff
@@ -0,0 +1,7 @@
+Args = --report-unsupported-elements-at-runtime \
+       --initialize-at-run-time=io.netty.channel.epoll.Epoll,io.netty.channel.epoll.Native,io.netty.channel.epoll.EpollEventLoop,io.netty.channel.epoll.EpollEventArray,io.netty.channel.unix.Errors,io.netty.channel.unix.IovArray,io.netty.channel.unix.Socket,org.apache.commons.logging.LogFactory \
+       --initialize-at-build-time=org.apache.http.pool.ConnPoolControl,org.apache.http.HttpClientConnection,org.apache.http.conn.routing.HttpRoute,org.apache.http.conn.HttpClientConnectionManager,org.apache.http.protocol.HttpContext,org.apache.http.conn.ConnectionRequest,org.apache.http.client.config.RequestConfig,org.apache.http.client.config.RequestConfig$Builder \
```

Comment:
Are these classes on the classpath by default for the lambda runtime?

Reply:
They are when you include the AWS SDK, eg:

```groovy
implementation platform("software.amazon.awssdk:bom:$awsSdkVersion")
implementation "software.amazon.awssdk:s3"
implementation "software.amazon.awssdk:rekognition"
```

Reply:
Then we would need these applied to new modules since if they are not on the classpath these entries will break the custom runtime on Graal